data/strings/strings.txt

```diff
@@ -16542,8 +16542,16 @@
     zh-Hans = 给 OSM 社区发送标记
     zh-Hant = 給 OSM 社區發送標記
 
+  [editor_note_hint]
+    tags = android
+    en = Describe map errors or additional details that cannot be edited with Organic Maps (optional)
```

Comment:
1. Maybe remove `(optional)` from here and write `Optional note to OpenStreetMap volunteers` in the title?
2. What about a shorter version?
```
    en = Describe errors on the map or things that cannot be edited with Organic Maps
    af = Beskryf foute in die kaart of wat nie met Organiese Kaarte geredigeer kan word nie
    ar = وصف الأخطاء على الخريطة أو الأشياء التي لا يمكن تحريرها باستخدام الخرائط العضوية
    az = Xəritədəki səhvləri və ya Üzvi Xəritələrdən istifadə edərək redaktə edilə bilməyənləri təsvir edin
    be = Апішыце памылкі на карце ці тое, што нельга рэдагаваць з дапамогай Organic Maps
    bg = Описване на грешки в картата или неща, които не могат да бъдат редактирани с Organic Maps
    ca = Descriu errors al mapa o què no es pot editar amb Mapes orgànics
    cs = Popište chyby na mapě nebo věci, které nelze upravit pomocí aplikace Organic Maps.
    da = Beskriv fejl på kortet eller ting, der ikke kan redigeres med Organic Maps
    de = Beschreiben Sie Fehler auf der Karte oder Dinge, die nicht mit Organic Maps bearbeitet werden können
    el = Περιγράψτε λάθη στο χάρτη ή πράγματα που δεν μπορούν να επεξεργαστούν με τους Organic Maps
    es = Describa los errores en el mapa o las cosas que no se pueden editar con Organic Maps
    et = Kirjeldage kaardil olevaid vigu või asju, mida ei saa Organic Mapsiga muuta.
    eu = Deskribatu mapan akatsak edo mapa organikoak erabiliz editatu ezin dena
    fa = خطاهای موجود در نقشه یا مواردی که با استفاده از نقشه های ارگانیک قابل ویرایش نیستند را شرح دهید
    fi = Kuvaile kartassa olevia virheitä tai asioita, joita ei voi muokata Organic Mapsilla.
    fr = Décrivez les erreurs sur la carte ou les éléments qui ne peuvent pas être modifiés avec Organic Maps.
    he = תאר שגיאות במפה או מה שלא ניתן לערוך באמצעות מפות אורגניות
    hi = मानचित्र में त्रुटियों का वर्णन करें या ऑर्गेनिक मानचित्र का उपयोग करके क्या संपादित नहीं किया जा सकता है
    hu = Írja le a térképen található hibákat vagy olyan dolgokat, amelyeket nem lehet az Organic Maps segítségével szerkeszteni.
    id = Jelaskan kesalahan pada peta atau hal-hal yang tidak dapat diedit dengan Peta Organik
    it = Descriva gli errori sulla mappa o le cose che non possono essere modificate con Organic Maps
    ja = 地図上のエラーやOrganic Mapsで編集できないものについて説明します。
    ko = 지도의 오류 또는 오가닉 맵으로 편집할 수 없는 항목에 대해 설명합니다.
    lt = Aprašykite žemėlapyje esančias klaidas arba dalykus, kurių negalima redaguoti naudojant "Organic Maps
    mr = नकाशामधील त्रुटी किंवा सेंद्रिय नकाशे वापरून काय संपादित केले जाऊ शकत नाही याचे वर्णन करा
    nb = Beskriv feil på kartet eller ting som ikke kan redigeres med Organic Maps
    nl = Beschrijf fouten op de kaart of dingen die niet bewerkt kunnen worden met Organic Maps
    pl = Proszę opisać błędy na mapie lub rzeczy, których nie można edytować za pomocą Organic Maps.
    pt = Descreva erros no mapa ou coisas que não podem ser editadas com o Organic Maps
    ro = Descrieți erorile de pe hartă sau lucrurile care nu pot fi editate cu Organic Maps
    ru = Опишите ошибки на карте или то, что нельзя редактировать с помощью Organic Maps
    sk = Popíšte chyby na mape alebo veci, ktoré sa nedajú upraviť pomocou aplikácie Mapy Organic
    sv = Beskriv fel på kartan eller saker som inte kan redigeras med Organic Maps
    sw = Eleza hitilafu kwenye ramani au kile ambacho hakiwezi kuhaririwa kwa kutumia Ramani za Kikaboni
    th = อธิบายข้อผิดพลาดในแผนที่หรือสิ่งที่ไม่สามารถแก้ไขได้โดยใช้แผนที่ทั่วไป
    tr = Haritadaki hataları veya Organik Haritalar ile düzenlenemeyen şeyleri tanımlayın
    uk = Опишіть помилки на карті або те, що не можна редагувати за допомогою Organic Maps
    vi = Mô tả lỗi trên bản đồ hoặc những gì không thể chỉnh sửa bằng Bản đồ không phải trả tiền
    zh-Hans = 描述地图上的错误或无法用有机地图编辑的内容
    zh-Hant = 描述地圖中的錯誤或無法使用有機地圖編輯的內容
```

Reply:
The shorter version is good. For the 'optional' I prefer the version after the hint in brackets, it stands out more that way.

Reply:
The third option "Note to OpenStreetMap volunteers (optional)" is also good, so let's proceed with that one